e2e/tests/121-Portal/MakeNewPayment/Safaricom/MakeFailedPaymentSafaricom.spec.ts

```diff
@@ -28,15 +28,19 @@ const paymentFilter =
 const paymentErrorMessages =
   'Error Occurred - Invalid Access Token - mocked_access_token';
 
+const registrationsSafaricomFailed = JSON.parse(
```

Comment:
is this still the best way to do this, and not 'structuredClone' or something like that?